src/main/java/uk/gov/hmcts/reform/sscs/model/lov/ref/data/CaseSubType.java

```diff
@@ -0,0 +1,1385 @@
+    BBA3_099_LIBERTY_TO_APPLY("BBA3-099LA", "LIBERTY TO APPLY", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_LATE_EXTENDING_BACK("BBA3-099LE", "LATE (EXTENDING BACK)", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_LABOUR_MARKET_ISSUES("BBA3-099LM", "LABOUR MARKET ISSUES", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_LIVING_TOGETHER("BBA3-099LT", "LIVING TOGETHER", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_MOBILITY_DLA("BBA3-099MD", "MOBILITY (DLA)", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_MINIMUM_INCOME_GUARANTEE("BBA3-099MG", "MINIMUM INCOME GUARANTEE", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_OVERPAYMENT_CAPITAL("BBA3-099OC", "OVERPAYMENT - CAPITAL", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_ONE_PROJECT("BBA3-099ON", "ONE PROJECT", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_OVERPAYMENT_STRAIGHTFORWARD("BBA3-099OS", "OVERPAYMENT - STRAIGHTFORWARD", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_OVERPAYMENT_COMPLEX("BBA3-099OX", "OVERPAYMENT - COMPLEX", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_PROPERTY_CAPITAL_SETTLEMENT("BBA3-099PC", "PROPERTY/CAPITAL SETTLEMENT", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_STAYED_PDD1("BBA3-099PD", "STAYED PDD1", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_PERSONS_FROM_ABROAD("BBA3-099PF", "PERSONS FROM ABROAD", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_PREMIUMS("BBA3-099PR", "PREMIUMS", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_RATE_OF_ASSESSMENT_PAYABILITY_ISSUES("BBA3-099RA", "RATE OF ASSESSMENT/PAYABILITY ISSUES", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_REDUCED_BENEFIT_DIRECTION("BBA3-099RB", "REDUCED BENEFIT DIRECTION", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_RATE_OF_ASSESSMENT_PAYABILITY_ISSUES_COMPLEX(
+        "BBA3-099RC",
+        "RATE OF ASSESSMENT/PAYABILITY ISSUES - COMPLEX",
+        OTHERS_EXTINCT_RARE_BENEFITS
+    ),
+    BBA3_099_RECIPROCAL_AGREEMENT("BBA3-099RG", "RECIPROCAL AGREEMENT", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_STAYED_CASES("BBA3-099SD", "STAYED CASES", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_FUNERAL("BBA3-099SF", "FUNERAL", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_MATERNITY("BBA3-099SM", "MATERNITY", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_SELF_EMPLOYED_STRAIGHTFORWARD("BBA3-099SS", "SELF EMPLOYED - STRAIGHTFORWARD", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_SUSPENSIONS("BBA3-099SU", "SUSPENSIONS", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_SELF_EMPLOYED_COMPLEX("BBA3-099SX", "SELF EMPLOYED - COMPLEX", OTHERS_EXTINCT_RARE_BENEFITS),
+    BBA3_099_STAYED_WIDOWS_BENEFIT("BBA3-099WB", "STAYED WIDOW'S BENEFIT", OTHERS_EXTINCT_RARE_BENEFITS);
+
+    private final String key;
+    private final String value;
+    private final CaseType caseType;
+
+    CaseSubType(String key, String value, CaseType caseType) {
+        this.key = key;
+        this.value = value;
+        this.caseType = caseType;
+    }
+
+
+    public static CaseSubType getCaseSubTypeBySubTypeValueAndCaseType(String value, CaseType caseType) {
```

Comment:
This method name is a bit confusing change to.. `getCaseSubTypeByValueAndCaseType` ? or I'd prefer just `getCaseSubType` the parameters tell you what you need to pass in.